internal/selfdescribe/monitors.go

```diff
@@ -1,24 +1,27 @@
 package selfdescribe
 
 import (
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
 	"go/doc"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"reflect"
 	"sort"
-	"strconv"
-	"strings"
-
-	log "github.com/sirupsen/logrus"
 
 	"github.com/signalfx/signalfx-agent/internal/monitors"
 )
 
+// Monitor metadata file.
+const monitorMetadataFile = "metadata.yaml"
+
 type metricMetadata struct {
 	Name        string `json:"name"`
 	Type        string `json:"type"`
 	Description string `json:"description"`
-	IsCustom    bool   `json:"isCustom" default:"true"`
+	Included    bool   `json:"included" default:"false"`
```

Comment:
@mstumpfx is "included" the best name for the whitelisted metrics in built-in content or do you know of a better term?